webviz_subsurface/plugins/_reservoir_simulation_timeseries.py

```diff
@@ -615,7 +615,11 @@ def _update_graph(
             # Titles for subplots
             titles = []
             for vec in vectors:
-                unit_vec = vec.lstrip("AVG_").lstrip("INTVL_")
+                unit_vec = (
+                    vec[4:]
+                    if vec.startswith("AVG_")
+                    else (vec[6:] if vec.startswith("INTVL_") else vec)
+                )
```

Comment:
Maybe we can change to this in order to both increase readability, but also to remember that this can be simplified when (in some years) Python 3.8 support is dropped.
```suggestion
                if sys.version_info >= (3.9):
                    unit_vec = vec.removeprefix("AVG_").removeprefix("INTVL_")
                else:
                    unit_vec = (
                        vec[4:]
                        if vec.startswith("AVG_")
                        else (vec[6:] if vec.startswith("INTVL_") else vec)
                    )
```